assignment-6&7/src/Components/Team Component/TeamMember.js

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+import { useParams } from 'react-router-dom';
+// import TeamContainer from "./CardContainer";
+import './style.css';
+
+const TeamMember = () => {
+    let { login } = useParams();
+    // console.log(login);
+
+    const [ member, setMember ] = useState({});
+    const { name, avatar_url, followers, following, bio, public_repos } = member;
+    // console.log(member);
+
+    const fetchTeamMember = async () => {
+        const response = await fetch(`https://api.github.com/users/${login}`);
+        const data = await response.json();
+        console.log(data);
+        setMember(data);
+    }
+
+    useEffect(() => {
+        fetchTeamMember();
+    }, []);
+
+    return (
+        <div className='team-row'>
```

Comment:
can use underscores while naming classes. We have a benefit that if they are too long then you can press ``command D + command c`` on VS CODE and it copies taking all of the class. Using a hyphen "-", you have to do all of the copying in the hard way.